build/common.sh

```diff
@@ -189,7 +189,7 @@ function kube::build::verify_prereqs() {
 
 function kube::build::docker_available_on_osx() {
   if [[ -z "${DOCKER_HOST}" ]]; then
-    if [[ -S "/var/run/docker.sock" ]]; then
+    if [[ -S "/var/run/docker.sock" ]] || [[ -S "${HOME}/.colima/docker.sock" ]]; then
       kube::log::status "Using Docker for MacOS"
```

Comment:
this log message is somewhat misleading 🤔 
maybe docker _on_ macOS?

Reply:
Certainly it will be convenient for colima users.
But the path depends on the tool used.
I seem it's appropriate to set the DOCKER_HOST environment variable in advance rather than adding the path for a particular tool into the source.

Reply:
@t-inu 
colima works over socket file and this is the default setup. 
Don't think colima works over docker_hos tcp and this will make the user environment to setup in way that brakes their loca setup using volumes

Reply:
I can put something more generic and use “docker context inspect” and parse the output with “jq” Can I assume that user has “jq”?